<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>register_company</class>
 <widget class="QWidget" name="register_company">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>register_company</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border:none;</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>901</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Image/background.jpg);</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>471</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;
border-radius:16px;
background:rgba(0,0,0,0.3);
color:#fff</string>
    </property>
    <widget class="QPushButton" name="register_login">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">display: inline-block;
 padding: 6px 12px;
		    margin-bottom: 0;
		    font-size: 14px;
		    font-weight: 400;
		    line-height: 1.42857143;
		    text-align: center;
		    white-space: nowrap;
		    vertical-align: middle;
		    -ms-touch-action: manipulation;
		    touch-action: manipulation;
		    cursor: pointer;
		    user-select: none;
		    background-image: none;
		    border: 1px solid transparent;
		    border-radius: 4px;
			color: #fff;
		    background-color: #31b0d5;
		    border-color: #269abc;
border-image:none;
</string>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="register_company_name">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
border: 1px solid #4288CE;
border-image:none;
color:#fff;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>请输入您公司的名字</string>
     </property>
    </widget>
    <widget class="QPushButton" name="register_new_company">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>200</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:none;
border:1px solid rgba(0,0,0,0);
border-radius:4px;
background:rgba(0,0,0,0.25);
color: #fff;
background:rgba(255,255,255,0);

</string>
     </property>
     <property name="text">
      <string>注册新公司</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="register_forget_company_name">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:none;
border:1px solid rgba(0,0,0,0);
border-radius:4px;
background:rgba(0,0,0,0.25);
color: #fff;
background:rgba(255,255,255,0);</string>
     </property>
     <property name="text">
      <string>忘记公司名称</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <zorder>register_company_name</zorder>
    <zorder>register_new_company</zorder>
    <zorder>register_forget_company_name</zorder>
    <zorder>register_login</zorder>
   </widget>
   <widget class="QPushButton" name="register_choose_login_way">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:25px;
background:#31b0d5;
color:#fff;
border-image:none;</string>
    </property>
    <property name="text">
     <string>界面</string>
    </property>
   </widget>
   <widget class="QLabel" name="register_company_message">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>460</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 170, 127);
border-image:none;</string>
    </property>
    <property name="text">
     <string>2682178536@qq.com客户QQ：2682178536</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
